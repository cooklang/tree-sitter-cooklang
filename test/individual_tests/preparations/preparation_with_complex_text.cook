Add @tomatoes{4}(washed, cored, and cut into 1/4-inch slices).