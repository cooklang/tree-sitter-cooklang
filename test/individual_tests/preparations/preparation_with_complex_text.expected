(recipe 
  (step 
    (ingredient 
      (name) 
      (amount (quantity))
      note: (note_text)))